cpp-low-level/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# syntax=docker.io/docker/dockerfile:1.4
+FROM --platform=linux/riscv64 riscv64/ubuntu:22.04 as builder
+
+RUN <<EOF
+apt-get update
+apt-get install -y --no-install-recommends autoconf automake ca-certificates curl build-essential libtool wget 
+rm -rf /var/lib/apt/lists/*
+EOF
+
+COPY --from=sunodo/sdk:0.1.0 /opt/riscv /opt/riscv
+WORKDIR /opt/cartesi/dapp
+COPY . .
+RUN make
+
+FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
+
+LABEL io.cartesi.rollups.sdk_version=0.1.0
```

Comment:
Isn't that supposed to be `io.sunodo.sdk_version=0.1.0`?

If that's the case, we could fix that on a different PR for all templates that are already there, including this one.

Reply:
yes. let's merge this and I open another PR